FordLabs/PeopleMover
# 136 calendar UI update (and new sub-nav)

## Issue
Resolves #136

## What was done
- [x] Create new subheader with calendar button, setting it up on feature tag '#newui'

## How to test
1. Open Peoplemover and navigate to a space. 
2. Note that subheading bar has not changed (narrower)
3. Change address bar, appending '#newui'. 
4. Updated sub-nav and calendar button will be visible.
